stats/HTTP.md

```diff
@@ -36,10 +36,10 @@ All client metrics should be tagged with the following.
 
 | Tag suffix         | Description                                                                                              |
 |--------------------|----------------------------------------------------------------------------------------------------------|
-| http.client_method | HTTP method, capitalized (i.e. GET, POST, PUT, DELETE, etc.)                                             |
```

Comment:
Why do we have separate tags for clients and servers? Client measurements will never be aggregated on a server views and vice versa.

Reply:
See in the grpc.md the FAQ section. I don't know where to put that section, maybe we should move it into a faq.md.